### PR TITLE
Update to_clarify.md

### PR DESCRIPTION
**Pull request by alvinlim540**
Merge #3  and latest questions

Please have a look.

### CODE REVIEW

File: docs/to_clarify.md [hunk 1]
```diff
@@ -18,16 +18,26 @@
...
     b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
 
     c. On the same note, what needs to be done during the checking-out process?
 
+    d. I would like to confirm the current traditional workflow of the check-in process.(Please fill me in if i got the procedure wrong)
+    Guest - Provide personal particulars, Photos of Passport/Identity Card, Booking Documents.?
+    
+    Front Desk staffs - Receiving the guest and collect relevant documents from guest. Do verification and give out hotels access card. Guide Guest to their room.
+
+
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Unnecessary new line

File: docs/to_clarify.md [hunk 2]
```diff
@@ -18,16 +18,26 @@
...
 1. Stated under the section "What Are We Looking For" that the solution should "verify travel and identification documents, booking documents, payment details, and make payment upon check-out, integrated with ICA (or other government stipulated system) for authentication and verification".
 
     a. Which government stipulated system is currently being used, and does the existing system provide API for our solution to communicate with it?
 
     b. Can we have several different samples of the above documents and the details of what are being verified for these documents?
 
+    c. What is the current architecture: Microservices or Monolith? 
+
+    d. How does the systems communicate with each other: REST API, SOAP, gRPC?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: I think this segment shouldn't be parked under my question, but it should be part of the first question under `Forum`:

> What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?

Let's remove

> What is the current architecture: Microservices or Monolith? 

I think it's not a relevant question for now. That's because I don't think we are expected to touch their code; we just have to integrate with them by calling the relevant API. So we don't need to be concerned with their architecture. Sorry for asking you to write this down in my previous comment.

> How does the systems communicate with each other: REST API, SOAP, gRPC?

Similarly for this, this question as it is isn't relevant. We should be asking how should our solution integrate with the existing solutions. Using REST, SOAP, gRPC or others?

File: docs/to_clarify.md [hunk 3]
```diff
@@ -18,16 +18,26 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    a. What needs to be done during the digital checking-in process? Are there any particular details/ workflow where it must be adhered to provide a check-in process?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Actually, I think this question can be removed. That's because we've already asked them above to 

> run through with us the current check-in/out process

File: docs/to_clarify.md [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
 
-1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems?
+1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems? If there is documentation of these systems, it would be easier for integration.
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: I think what you added here isn't meaningful because, duh, documentation definitely helps in integration.

What we should be adding is: 

> If so, what are the APIs used to communicate between these systems? **Will documentation & assistance be provided if we face certain issues interfacing with these systems?**

This sentence should also be merged with the above point

> What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?

File: docs/to_clarify.md [hunk 5]
```diff
@@ -18,16 +18,26 @@
...
+    a. What needs to be done during the digital checking-in process? Are there any particular details/ workflow where it must be adhered to provide a check-in process?
 
     b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
 
     c. On the same note, what needs to be done during the checking-out process?
 
+    d. I would like to confirm the current traditional workflow of the check-in process.(Please fill me in if i got the procedure wrong)
+    Guest - Provide personal particulars, Photos of Passport/Identity Card, Booking Documents.?
+    
+    Front Desk staffs - Receiving the guest and collect relevant documents from guest. Do verification and give out hotels access card. Guide Guest to their room.
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md]: Actually, I think this question can be removed. That's because we've already asked them above to

> run through with us the current check-in/out process

**Review by Zhiyuan-Amos — CHANGES_REQUESTED** [review]
Can you also add:

1. https://github.com/openinnovation-hotel-check-in-out/ideation/pull/3#discussion_r462925433
2. https://github.com/openinnovation-hotel-check-in-out/ideation/pull/3#discussion_r462713479 the part on asking about existing Digital Concierges

Also, I noticed that you didn't add in some of the questions that we discussed previously e.g. something about Booking Engine. I can't exactly remember what exactly are these questions, are they no longer relevant that's why you've omitted them?

File: docs/to_clarify.md (hunk 1 above)
Comment (alvinlim540) [reply docs/to_clarify.md]: i see that it is not relevant anymore. That's why i didnt add in.  The focus was asking about API and not inferring further about their booking Engine.

File: docs/to_clarify.md (hunk 2 above)
Comment (alvinlim540) [reply docs/to_clarify.md]: It's okay sir. If there is a need, we will put it back. Will remove and update.

File: docs/to_clarify.md (hunk 3 above)
Comment (alvinlim540) [reply docs/to_clarify.md]: Are you referring to the forum question on 'pain points' or safe and secured? My intention was to cover missing requirements from their POV.

File: docs/to_clarify.md (hunk 1 above)
Comment (Zhiyuan-Amos) [reply docs/to_clarify.md]: I think your above message is written in response to my comment https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#pullrequestreview-459692826. In that case, you shouldn't reply on this thread. Rather, you should reply by creating a new comment below and quoting the relevant segments.

File: docs/to_clarify.md (hunk 5 above)
Comment (alvinlim540) [reply docs/to_clarify.md]: okay noted.

File: docs/to_clarify.md (hunk 3 above)
Comment (Zhiyuan-Amos) [reply docs/to_clarify.md]: The question on 'pain points'. @ahyong92 what do you think: is this question still needed?

File: docs/to_clarify.md (hunk 4 above)
Comment (alvinlim540) [reply docs/to_clarify.md]: I dont see anything to add anymore. It is concise enough.

Comment (Zhiyuan-Amos) [reply docs/to_clarify.md]: I think that we should ask whether they will provide assistance to create additional API for us in the event what we need isn't provided by the existing systems. Or do we have to inherit their code base and implement the API on our own.

Comment (alvinlim540) [reply docs/to_clarify.md]: It will be great if they provide assistance. This would cut time rather than dig from scratch. Good point to note sir!
 will add in.

File: docs/to_clarify.md (hunk 3 above)
Comment (gohyk) [reply docs/to_clarify.md]: I don't think we should remove, it's something we have to keep in mind when they run through with us the process.

Comment (Zhiyuan-Amos) [reply docs/to_clarify.md]: I would have thought that "what needs to be done during the check in process" would have been covered as they explained about their check-in process :P 

But I'm ok either way. Just that Alvin's addition is slightly unclear and can be better reworded.

Comment (gohyk) [reply docs/to_clarify.md]: Oh I get your point. @alvinlim540, I guess we should just remove part "a" from this point then?

Comment (alvinlim540) [reply docs/to_clarify.md]: @Zhiyuan-Amos @ahyong92 I also think that my sentence is not specific enough. 
"a. What needs to be done during the digital checking-in process? Is there any of the hidden/back-end process must be adhered to provide a seamless and contact-less check-in process?"

File: docs/to_clarify.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
 
-1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems?
+1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md:9]: Unnecessary change

File: docs/to_clarify.md [hunk 7]
```diff
@@ -18,7 +18,7 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    What needs to be done during the digital checking-in process? Is there any of the hidden/back-end process must be adhered to provide a seamless and contact-less check-in process?
```
Comment (Zhiyuan-Amos) [inline docs/to_clarify.md:21]: Why is the bullet point a. removed? 

> Is there any of the hidden/back-end process must be adhered to

What does this mean? As in, what an example of a hidden/back-end process?

**Review by Zhiyuan-Amos — COMMENTED** [review]
These weren't addressed:

1. https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#discussion_r464177779
1. https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#pullrequestreview-459692826

File: docs/to_clarify.md (hunk 7 above)
Comment (alvinlim540) [reply docs/to_clarify.md:21]: An unknown step for them to explain. IMO i was thinking of collecting a list of tenants whom are currently staying? Or maybe steps where it is not controllable by software.

Comment (Zhiyuan-Amos) [reply docs/to_clarify.md:21]: I don't think it's necessary. As mentioned above https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#discussion_r464434737, we can remove this point entirely. This question might seem good-to-have for now, but as we clarify User Stories with them, we would necessarily get the answer for this question already. So, let's remove it.
